packages/ibm-products/src/components/EditInPlace/EditInPlace.tsx

```diff
@@ -248,16 +248,21 @@ export let EditInPlace = forwardRef<HTMLDivElement, EditInplaceProps>(
       onCancelHandler();
     };
 
+    const removeFocus = () => {
```

Comment:
Hey @AlexanderMelox ,
Changes look fine. Just one doubt.
 I could see on `escape` press,  `inputRef.current?.blur();` and `setFocused(false);` was already happening previously. Inside `escapeHandler`, they are setting `false` to `setFocused`. I was just wondering why this issue existed earlier on `Escape` key.

Reply:
Okay I see, I realized that the focused state was only being triggered on enter when canSave is true. But for consistency, I'm going to only do it on the removeFocus function to keep the code DRY

Reply:
Thank you!